internal/captain/command.go

```diff
@@ -416,7 +417,7 @@ func (c *Command) runner(cobraCmd *cobra.Command, args []string) error {
 	err := c.execute(c, args)
 	if !c.deferAnalytics {
 		exitCode := errs.UnwrapExitCode(failures.ToError(err))
-		analytics.EventWithValue(analytics.CatCommandExit, subCommandString, int64(exitCode))
+		analytics.EventWithLabel(analytics.CatCommandExit, subCommandString, strconv.Itoa(exitCode))
```

Comment:
Does `EventWithValue` not work at all?

Reply:
It does, I was mistaken. There's 4 levels: category, action, label, value. Value isn't really useful for exit codes, since we want to group by these results and "Value" fields aren't really made for that.

Reply:
Ah! That makes sense...